_data/videos.yaml

```diff
@@ -3,6 +3,13 @@ description: "South Africa's National and Provincial budget data from National T
 selected_tab: learning-centre
 selected_sidebar: videos
 data:
+- id: mtbps
```

Comment:
Will this mean it will show up in the video pages, too?

Reply:
Another option is just to put the iframe into the page directly. That's what I initially had in mind, I hadn't thought of this mechanism.

Reply:
Yes.

Reply:
The iframe isn't responsive by default, so I'll have to add some CSS hacking to it (which is already done in the video component). Personally I don't see a problem with it being in the videos section.

How strongly do you feel about this?

Reply:
This is fine.